content/en/docs/setup/install/istioctl/index.md

```diff
@@ -59,10 +59,33 @@ Various settings can be configured to modify the installations. For example, to
 $ istioctl install --set meshConfig.accessLogFile=/dev/stdout
 {{< /text >}}
 
+{{< tip >}}
+Many of the examples on this page and elsewhere in the documentation are written using --set to modify installation
+parameters, rather than passing a configuration file with -f. This is done to make the examples more compact.
+The two methods are equivalent, but -f is strongly recommended for production. The above command would be written as
+follows using -f:
+
+{{< text yaml >}}
+# my-config.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    accessLogFile: /dev/stdout
+{{< /text >}}
+
+{{< text bash >}}
+$ istioctl install -f my-config.yaml
+{{< /text >}}
+
+{{< /tip >}}
+
+{{< tip >}}
+The full API is documented in the [`IstioOperator` API reference](/docs/reference/config/istio.operator.v1alpha1/).
 In general, you can use the `--set` flag in `istioctl` as you would with
-Helm. The only difference is you must
-prefix the setting paths with `values.` because this is the path to the Helm pass-through API in the
-[`IstioOperator` API](/docs/reference/config/istio.operator.v1alpha1/).
+Helm, and the Helm `values.yaml` API is currently supported for backwards compatibility. The only difference is you must
```

Comment:
Will be good to add reference for helm installation from 1.5 docs.

Reply:
i'd rather leave this as is since pointing to helm may encourage use of that api.